packages/app/src/debug/guide/GuideCard1.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <PromoCard
+    :title="t('debugPage.emptyStates.slideshow.step1.title')"
+    :body="t('debugPage.emptyStates.slideshow.step1.description')"
+  >
+    <template #image>
+      <Illustration />
+    </template>
+    <template #action>
```

Comment:
This looks generally fine, but one other options might to be have 1 components and define the options in an array, like:

```
const guideCards = [
  {
      action: goForward,
      title: t('...'),
      leftLabel: ...
  }
]

// ...
<GuideCard v-for="card of guideCards" :body="card.body" />
// ... or ...
<GuideCard v-for="card of guideCards" :card="card" />
```

Not fussed, but might be a good way to save a bit of duplication. WDYT?

Reply:
This is how `Slideshow` was originally structured, but I backed away from that since there are now a number of different permutations for slides. It got unweildy to map all those thru via `props`, so I ended up with different components to keep the props manageable (and testable). I'll do a little restructuring to get the cards better managed and we can see how it feels at that point